sdk/lib/rtl/amd64/except.c

```diff
@@ -20,57 +20,37 @@ RtlRaiseException(IN PEXCEPTION_RECORD ExceptionRecord)
 {
     CONTEXT Context;
     NTSTATUS Status = STATUS_INVALID_DISPOSITION;
-    ULONG64 ImageBase;
-    PRUNTIME_FUNCTION FunctionEntry;
-    PVOID HandlerData;
-    ULONG64 EstablisherFrame;
 
-    /* Capture the context */
+    /* Capture the current context */
     RtlCaptureContext(&Context);
 
-    /* Get the function entry for this function */
-    FunctionEntry = RtlLookupFunctionEntry(Context.Rip,
-                                           &ImageBase,
-                                           NULL);
+    /* Fix up Context.Rip for the caller */
+    Context.Rip = (ULONG64)_ReturnAddress();
 
-    /* Check if we found it */
-    if (FunctionEntry)
+    /* Fix up Context.Rsp for the caller */
+    Context.Rsp = (ULONG64)_AddressOfReturnAddress() + 8;
```

Comment:
Explaining why "+8" is needed? (aka. why this fixing)

Reply:
Bruh, it's literally in the comment.